superset/README.md

```diff
@@ -117,3 +117,35 @@ superset fab export-roles --path roles/roles.json
 ```
 
 I've noticed the importing can sometimes take a few minutes.
+
+## How to programmatically create PUDL table dashboards
+
+We've designed a dashboard template for filtering and downloading PUDL tables. The `./automation/create_table_dashboards.py` script
+programmatically creates one of these dashboards and all the charts it depends on.
+
+To use this script you'll first need to assign the `superset-bot` credentials to env vars:
+
+```
+export SUPERSET_USERNAME=superset-bot
+export SUPERSET_PASSWORD={grab password from Google Secrets}
+```
+
+Then, to create a dashboard, run:
+
+```
+python ./automation/create_table_dashboards.py [TABLE_NAMES]...
+```
+
+### Limitations / Open questions
```

Comment:
This is deserving of a more thorough right up which we can do later if we want to move superset into production.